source/agora/consensus/data/ConsensusParams.d

```diff
@@ -80,10 +89,11 @@ public immutable class ConsensusParams
 
     /// Default for unittest, uses the test genesis block
     version (unittest) public this (
+        in PublicKey commons_budget_address = CommonsBudgetKeyPair.address,
```

Comment:
Move this parameter last and you won't have to modify any other tests.

Reply:
No, don't make it a parameter. The Genesis block is not a parameter either.

Reply:
Right good point, this is for tests only.

Reply:
Yeah, I'll remove it from the parameters, and I'll do the same thing as the Genesis block.